app/services/work_packages/copy_service.rb

```diff
@@ -77,8 +77,7 @@ def copied_attributes(work_package, override)
     attributes = work_package
                    .attributes
                    .slice(*writable_work_package_attributes(work_package))
-                   .merge("parent_id" => work_package.parent_id,
-                          "custom_field_values" => work_package.custom_value_attributes)
```

Comment:
parent_id is removed here because it is already part of the sliced params?

Reply:
Yes, that's something we realized with Jens in a pairing session.
We did not understand why this was here as it seemed it would be picked by the `slice` call above. So we looked at tests, tried with, and without, and as expected the parent is carried over, so we removed it.